src/Extractor/ContentExtractor.php

```diff
@@ -669,4 +686,51 @@ private function extractBody($detectBody, $xpathExpression, \DOMNode $node, $typ
 
         return false;
     }
+
+    /**
+     * Extract and apply a callback to an entity according to a pattern
+     *
+     * @param string $entity            Entity to look for
+     * @param string $pattern           Pattern to look for
+     * @param callable $returnCallback  Function to apply on the value
+     *
+     * @return bool Telling if the entity has been found
+     */
+    private function extractEntityFromPattern($entity, $pattern, $returnCallback = null)
+    {
+        // we define the default callback here
+        if (!is_callable($returnCallback)) {
+            $returnCallback = function($e) {
+                return $e;
```

Comment:
Looks like it'll be `trim()` most the time, I think we can use `trim()` as the default one.